src/plone/restapi/serializer/nextprev.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from Acquisition import aq_inner
+from Acquisition import aq_parent
+from plone.app.layout.nextprevious.interfaces import INextPreviousProvider
+
+
+class NextPrevious(object):
+    """Facade with more pythonic interface"""
+
+    def __init__(self, context):
+        self.context = context
+        parent = aq_parent(aq_inner(context))
+        self.adapter = INextPreviousProvider(parent, None)
+
+    @property
+    def _enabled(self):
+        if self.adapter is None:
+            return False
+        return self.adapter.enabled
+
+    @property
+    def next(self):
+        """ return info about the next item in the container """
+        if not self._enabled:
+            return {}
+        data = self.adapter.getNextItem(self.context)
+        if data is None:
+            return {}
+        return {
+            "@id": data["url"].lstrip('/view'),
```

Comment:
It's unfortunate that the adapter adds the /view then we have to strip it :(

Reply:
yeah..

Reply:
yeah..